.travis.yml

```diff
@@ -32,6 +32,16 @@ before_install:
 
 script:
   - mvn -B test
+  # Run Docker-in-Docker tests
+  # Yes, we use selenium image, but we use it anyway in BrowserContainer's tests
+  - >
+      DOCKER_HOST=unix:///var/run/docker.sock DOCKER_TLS_VERIFY= docker run --rm \
+      -v "$HOME/.m2":/root/.m2/ \
+      -v /var/run/docker.sock:/var/run/docker.sock \
+      -v "$(pwd)":"$(pwd)" \
+      -w "$(pwd)" \
+      selenium/standalone-chrome-debug:2.52.0 \
+      ./mvnw -B -pl core test -Dtest=*GenericContainerRuleTest
```

Comment:
GenericContainerRuleTest covers most of the cases. Right now we can't use all the tests because some of them make incorrect assumptions